oak_functions_containers_app/src/lib.rs

```diff
@@ -0,0 +1,95 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use crate::proto::oak::functions::{
+    oak_functions_server::{OakFunctions, OakFunctionsServer},
+    AbortNextLookupDataResponse, Empty, ExtendNextLookupDataRequest, ExtendNextLookupDataResponse,
+    FinishNextLookupDataRequest, FinishNextLookupDataResponse, InitializeRequest,
+    InitializeResponse, InvokeRequest, InvokeResponse,
+};
+use anyhow::anyhow;
+use tokio::net::TcpListener;
+use tokio_stream::wrappers::TcpListenerStream;
+
+mod proto {
+    pub mod oak {
+        pub mod functions {
+            #![allow(clippy::return_self_not_must_use)]
+            tonic::include_proto!("oak.functions");
+        }
+    }
+}
+
+// Instance of the OakFunctions service for Oak Containers.
+pub struct OakFunctionsContainersService {}
+
+// Temporarily allow unused varaibles for requests.
```

Comment:
You should prefer prefixing the unused variables with underscores instead.

Also it looks like they are actually used, at least for logging, so I'm surprised this is needed.

Reply:
I think you caught an outdated bit of code here. Removing the comment and trait.